### ISSUE TITLE
Conda build dependency conflict with rio-cogeo >=2.3.0 and fiona >=1.8.19

### ISSUE THREAD

**Issue by JoakimJoensuu** [conda-forge/rio-cogeo-feedstock#26]
<!--
Thanks for reporting your issue.
Please fill out the sections below.
-->

- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:

I am trying to build conda package with rio-cogeo >=2.3.0 and fiona >=1.8.19 dependencies.

However that seems to be impossible due to conflicts between those two. I was motivated to write this issue because fiona is used to read and write geographical data files and these two will definitely be used together in some projects.

Leaving version requirement out from one of those will install either `rio-cogeo 2.0.1 and fiona 1.8.20` or `rio-cogeo 2.3.1 and fiona 1.8.18`.

Is there any solutions for this? Or is the only option to wait for compatible future releases?

Installing with pip `pip install .` does not raise any warnings or errors.

<details><summary>conda build ./ -c conda-forge</summary>

```console
$ conda build ./ -c conda-forge
/home/user/miniconda3/lib/python3.8/site-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.26.5) or chardet (3.0.4) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.16
WARNING:conda_build.metadata:No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.16
Copying /home/user/Documents/test to /home/user/miniconda3/conda-bld/conflicting_1628601856491/work/
Adding in variants from internal_defaults
INFO:conda_build.variants:Adding in variants from internal_defaults
Attempting to finalize metadata for conflicting
INFO:conda_build.metadata:Attempting to finalize metadata for conflicting
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done
BUILD START: ['conflicting-1.0.0-build_string.tar.bz2']
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done

## Package Plan ##

  environment location: /home/user/miniconda3/conda-bld/conflicting_1628601856491/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh


The following NEW packages will be INSTALLED:

    _libgcc_mutex:    0.1-conda_forge           conda-forge
    _openmp_mutex:    4.5-1_gnu                 conda-forge
    ca-certificates:  2021.7.5-h06a4308_1                  
    certifi:          2021.5.30-py38h578d9bd_0  conda-forge
    ld_impl_linux-64: 2.36.1-hea4e1c9_2         conda-forge
    libffi:           3.3-h58526e2_2            conda-forge
    libgcc-ng:        11.1.0-hc902ee8_8         conda-forge
    libgomp:          11.1.0-hc902ee8_8         conda-forge
    libstdcxx-ng:     11.1.0-h56837e0_8         conda-forge
    ncurses:          6.2-h58526e2_4            conda-forge
    openssl:          1.1.1k-h7f98852_0         conda-forge
    pip:              21.2.3-pyhd8ed1ab_0       conda-forge
    python:           3.8.11-h12debd9_0_cpython            
    python_abi:       3.8-2_cp38                conda-forge
    readline:         8.1-h46c0cb4_0            conda-forge
    setuptools:       52.0.0-py38h06a4308_0                
    sqlite:           3.36.0-h9cd32fc_0         conda-forge
    tk:               8.6.10-h21135ba_1         conda-forge
    wheel:            0.36.2-pyhd3deb0d_0       conda-forge
    xz:               5.2.5-h516909a_1          conda-forge
    zlib:             1.2.11-h516909a_1010      conda-forge

Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.*, but conda is ignoring the .* and treating it as 1
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.*, but conda is ignoring the .* and treating it as 1
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.*, but conda is ignoring the .* and treating it as 3
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.*, but conda is ignoring the .* and treating it as 3
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.6.*, but conda is ignoring the .* and treating it as 3.6
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.6.*, but conda is ignoring the .* and treating it as 3.6
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 0.23.*, but conda is ignoring the .* and treating it as 0.23
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 0.23.*, but conda is ignoring the .* and treating it as 0.23
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.*, but conda is ignoring the .* and treating it as 3
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.*, but conda is ignoring the .* and treating it as 3
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 4.*, but conda is ignoring the .* and treating it as 4
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 4.*, but conda is ignoring the .* and treating it as 4

Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                                                                                                                                                                                                                              

Leaving build/test directories:
  Work:
 /home/user/miniconda3/conda-bld/work 
  Test:
 /home/user/miniconda3/conda-bld/test_tmp 
Leaving build/test environments:
  Test:
source activate  /home/user/miniconda3/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho 
  Build:
source activate  /home/user/miniconda3/conda-bld/_build_env 


Traceback (most recent call last):
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda_build/environ.py", line 802, in get_install_actions
    actions = install_actions(prefix, index, specs, force=True)
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda/plan.py", line 474, in install_actions
    txn = solver.solve_for_transaction(prune=prune, ignore_pinned=not pinned)
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda/core/solve.py", line 114, in solve_for_transaction
    unlink_precs, link_precs = self.solve_for_diff(update_modifier, deps_modifier,
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda/core/solve.py", line 157, in solve_for_diff
    final_precs = self.solve_final_state(update_modifier, deps_modifier, prune, ignore_pinned,
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda/core/solve.py", line 281, in solve_final_state
    ssc = self._run_sat(ssc)
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda/core/solve.py", line 815, in _run_sat
    ssc.solution_precs = ssc.r.solve(tuple(final_environment_specs),
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda/resolve.py", line 1322, in solve
    self.find_conflicts(specs, specs_to_add, history_specs)
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda/resolve.py", line 352, in find_conflicts
    raise UnsatisfiableError(bad_deps, strict=strict_channel_priority)
conda.exceptions.UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions

Package blas conflicts for:
rio-cogeo[version='>=2.3.0'] -> numpy~=1.15 -> blas[version='*|1.0|1.1|1.0',build='mkl|openblas|openblas|openblas']
fiona[version='>=1.8.19'] -> numpy[version='>=1.19.5,<2.0a0'] -> blas[version='*|1.0',build='mkl|openblas']

Package python conflicts for:
rio-cogeo[version='>=2.3.0'] -> click -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|3.4.*|>=3|>=3.0']
rio-cogeo[version='>=2.3.0'] -> python[version='>3.6']
python

Package libgdal conflicts for:
fiona[version='>=1.8.19'] -> gdal -> libgdal[version='2.2.4|2.2.4|2.3.0|2.3.2|2.3.2|2.3.3|2.4.1|2.4.1|2.4.1|2.4.2|2.4.2|2.4.2|2.4.2|2.4.2|2.4.3|2.4.3|2.4.4|2.4.4|3.0.1|3.0.1|3.0.1|3.0.1|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.3|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.1.0|3.1.0|3.1.0|3.1.0|3.1.1|3.1.1|3.1.1|3.1.2|3.1.2|3.1.2|3.1.2|3.1.2|3.1.3|3.1.3|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.2.0|3.2.1|3.2.1|3.2.1|3.2.1|3.2.1|3.2.1|3.2.1|3.2.1|3.2.1|3.2.1|3.2.2|3.2.2|3.2.2|3.2.2|3.2.2|3.2.2|3.2.2|3.2.2|3.3.0|3.3.0|3.3.0|3.3.0|3.3.1|>=2.2.2,<2.3.0a0',build='hc8d23f9_1|heea9cce_1|heea9cce_1|h9d4a965_0|heae24aa_8|ha50d65d_9|hc4f5fd6_10|hbef8c27_4|h04a3d72_6|h182c168_7|h182c168_8|h2b6fda6_0|h5439ffd_1|hcf75f94_9|h3d260b8_0|h30a29e3_5|hefdf094_6|h20022a4_8|h20022a4_0|h022d3c0_1|h9e6407a_4|hbd881e9_7|h99f9e9c_8|he6a97d6_10|h4ed0dcf_3|h6687615_0|hb2a6f5f_1|hf6fc319_2|hb2a6f5f_3|h670eac6_1|h50e41a3_1|h96b6e7a_3|h0a4f56a_5|hbde00c8_6|h804b7da_10|h78c8158_11|h679344c_14|h3215d8d_15|h8f005ca_17|h2812452_0|h52563cd_1|h744752d_3|hbde00c8_5|h804b7da_0|h78c8158_1|hbf32332_4|h8f005ca_7|hbf32332_0|h679344c_1|h4dcd70e_2|h8f005ca_3|h8f005ca_0|h8f005ca_1|h4dcd70e_6|h679344c_5|hbf32332_3|h928efbf_2|h804b7da_9|h6636813_8|h38ff51b_7|he6eed4e_6|h0a4f56a_4|h96b6e7a_2|h24f7f85_0|h4dcd70e_16|hbf32332_13|h928efbf_12|h6636813_9|h38ff51b_8|he6eed4e_7|h744752d_4|h02eeb80_2|h670eac6_0|h670eac6_0|h670eac6_4|h6687615_2|h6687615_1|h4ed0dcf_0|h0b4441a_2|h2e1b11c_1|he6a97d6_0|h99f9e9c_9|h3dfc09a_6|h4f4a046_5|h94bbfbd_3|hce44138_2|h20022a4_0|hbe56659_7|h852c1c0_4|h5534617_3|hc7cfd23_2|h3d260b8_1|h7c14f60_1|h27ab9cc_0|h3d260b8_10|hf47eb90_8|h2bea7cd_7|h2f07a13_9|hd53ac37_8|h524de44_5|h2e7e64b_0|hda2fc8e_0']
fiona[version='>=1.8.19'] -> libgdal[version='>=3.2.2,<3.3.0a0|>=3.3.0,<3.4.0a0']

Package _libgcc_mutex conflicts for:
fiona[version='>=1.8.19'] -> libgcc-ng[version='>=9.3.0'] -> _libgcc_mutex[version='*|0.1',build='conda_forge|main|main']
python -> libgcc-ng[version='>=9.3.0'] -> _libgcc_mutex[version='*|0.1',build='conda_forge|main|main']

Package numpy conflicts for:
fiona[version='>=1.8.19'] -> gdal -> numpy[version='1.10.*|1.11.*|1.12.*|1.13.*|>=1.11|>=1.11.3,<2.0a0|>=1.14.6,<2.0a0|>=1.16.5,<2.0a0|>=1.16.6,<2.0a0|>=1.19.4,<2.0a0|>=1.19.2,<2.0a0|>=1.9.3,<2.0a0|>=1.18.5,<2.0a0|>=1.8|>=1.9|>=1.7|1.9.*|1.8.*']
fiona[version='>=1.8.19'] -> numpy[version='>=1.17.5,<2.0a0|>=1.19.5,<2.0a0']

Package click conflicts for:
fiona[version='>=1.8.19'] -> click-plugins[version='>=1.0'] -> click[version='>=3.0|>=4.0']
fiona[version='>=1.8.19'] -> click[version='>=4.0,<8']The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.27=0
  - feature:|@/linux-64::__glibc==2.27=0
  - python -> libgcc-ng[version='>=9.3.0'] -> __glibc[version='>=2.17']

Your installed version is: 2.27


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/user/miniconda3/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda_build/cli/main_build.py", line 481, in main
    execute(sys.argv[1:])
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda_build/cli/main_build.py", line 470, in execute
    outputs = api.build(args.recipe, post=args.post, test_run_post=args.test_run_post,
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda_build/api.py", line 186, in build
    return build_tree(
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda_build/build.py", line 3068, in build_tree
    packages_from_this = build(metadata, stats,
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda_build/build.py", line 2108, in build
    create_build_envs(top_level_pkg, notest)
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda_build/build.py", line 1986, in create_build_envs
    raise e
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda_build/build.py", line 1965, in create_build_envs
    environ.get_install_actions(m.config.test_prefix,
  File "/home/user/miniconda3/lib/python3.8/site-packages/conda_build/environ.py", line 804, in get_install_actions
    raise DependencyNeedsBuildingError(exc, subdir=subdir)
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {"libgdal[version='>=3.2.2,<3.3.0a0|>=3.3.0,<3.4.0a0']", "blas[version='*|1.0',build='mkl|openblas']", "python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0|>=3.5,<3.6.0a0|3.4.*|>=3|>=3.0']", "python[version='>3.6']", "numpy[version='>=1.17.5,<2.0a0|>=1.19.5,<2.0a0']", "libgdal[version='2.2.4|2.2.4|2.3.0|2.3.2|2.3.2|2.3.3|2.4.1|2.4.1|2.4.1|2.4.2|2.4.2|2.4.2|2.4.2|2.4.2|2.4.3|2.4.3|2.4.4|2.4.4|3.0.1|3.0.1|3.0.1|3.0.1|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.2|3.0.3|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.0.4|3.1.0|3.1.0|3.1.0|3.1.0|3.1.1|3.1.1|3.1.1|3.1.2|3.1.2|3.1.2|3.1.2|3.1.2|3.1.3|3.1.3|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.1.4|3.2.0|3.2.1|3.2.1|3.2.1|3.2.1|3.2.1|3.2.1|3.2.1|3.2.1|3.2.1|3.2.1|3.2.2|3.2.2|3.2.2|3.2.2|3.2.2|3.2.2|3.2.2|3.2.2|3.3.0|3.3.0|3.3.0|3.3.0|3.3.1|>=2.2.2,<2.3.0a0',build='hc8d23f9_1|heea9cce_1|heea9cce_1|h9d4a965_0|heae24aa_8|ha50d65d_9|hc4f5fd6_10|hbef8c27_4|h04a3d72_6|h182c168_7|h182c168_8|h2b6fda6_0|h5439ffd_1|hcf75f94_9|h3d260b8_0|h30a29e3_5|hefdf094_6|h20022a4_8|h20022a4_0|h022d3c0_1|h9e6407a_4|hbd881e9_7|h99f9e9c_8|he6a97d6_10|h4ed0dcf_3|h6687615_0|hb2a6f5f_1|hf6fc319_2|hb2a6f5f_3|h670eac6_1|h50e41a3_1|h96b6e7a_3|h0a4f56a_5|hbde00c8_6|h804b7da_10|h78c8158_11|h679344c_14|h3215d8d_15|h8f005ca_17|h2812452_0|h52563cd_1|h744752d_3|hbde00c8_5|h804b7da_0|h78c8158_1|hbf32332_4|h8f005ca_7|hbf32332_0|h679344c_1|h4dcd70e_2|h8f005ca_3|h8f005ca_0|h8f005ca_1|h4dcd70e_6|h679344c_5|hbf32332_3|h928efbf_2|h804b7da_9|h6636813_8|h38ff51b_7|he6eed4e_6|h0a4f56a_4|h96b6e7a_2|h24f7f85_0|h4dcd70e_16|hbf32332_13|h928efbf_12|h6636813_9|h38ff51b_8|he6eed4e_7|h744752d_4|h02eeb80_2|h670eac6_0|h670eac6_0|h670eac6_4|h6687615_2|h6687615_1|h4ed0dcf_0|h0b4441a_2|h2e1b11c_1|he6a97d6_0|h99f9e9c_9|h3dfc09a_6|h4f4a046_5|h94bbfbd_3|hce44138_2|h20022a4_0|hbe56659_7|h852c1c0_4|h5534617_3|hc7cfd23_2|h3d260b8_1|h7c14f60_1|h27ab9cc_0|h3d260b8_10|hf47eb90_8|h2bea7cd_7|h2f07a13_9|hd53ac37_8|h524de44_5|h2e7e64b_0|hda2fc8e_0']", "_libgcc_mutex[version='*|0.1',build='conda_forge|main|main']", "blas[version='*|1.0|1.1|1.0',build='mkl|openblas|openblas|openblas']", "click[version='>=3.0|>=4.0']", "numpy[version='1.10.*|1.11.*|1.12.*|1.13.*|>=1.11|>=1.11.3,<2.0a0|>=1.14.6,<2.0a0|>=1.16.5,<2.0a0|>=1.16.6,<2.0a0|>=1.19.4,<2.0a0|>=1.19.2,<2.0a0|>=1.9.3,<2.0a0|>=1.18.5,<2.0a0|>=1.8|>=1.9|>=1.7|1.9.*|1.8.*']", 'feature:/linux-64::__glibc==2.27=0', 'feature:|@/linux-64::__glibc==2.27=0', "__glibc[version='>=2.17']"}

```

</details/>


I have following Python project as an example:
```
.
├── meta.yaml
├── package
│   ├── __init__.py
│   └── __main__.py
├── README.md
└── setup.py
```

**meta.yaml**
```
{% set data = load_setup_py_data(setup_file="./setup.py", from_recipe_dir=True, recipe_dir=RECIPE_DIR) %}

package:
  name: "{{ data['name'] }}"
  version: "{{ data['version'] }}"

source:
  path: ./

build:
  noarch: python
  number: 0
  string: build_string
  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt"
  entry_points:
    {% for script in data['entry_points']['console_scripts'] %}
    - {{ script }}
    {% endfor %}

requirements:
  build:
    - python
  host:
    - python
  run:
    - python
    {% for requirement in data['install_requires'] %}
    - {{ requirement.lower() }}
    {% endfor %}
```

**package/\_\_main\_\_.py**
```
import rio_cogeo
import fiona


def main():
    print("Works")
```

**README.md**
```
conda deactivate
conda build purge-all
conda create -n conda_env -y
conda activate conda_env
conda build ./ -c conda-forge
conda install --use-local conflicting -c conda-forge -y
conflicting
```

**setup.py**
```
from setuptools import find_packages, setup

setup(
    name="conflicting",
    version="1.0.0",
    description="Dependencies conflicting",
    python_requires=">=3.8",
    install_requires=[
        "fiona >=1.8.19",
        "rio-cogeo >=2.3.0",
    ],
    packages=find_packages(),
    entry_points={
        "console_scripts": [
            "conflicting = package.__main__:main",
        ]
    },
)

```

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /home/user/miniconda3/envs/conda_env:
#
# Name                    Version                   Build  Channel
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
/home/user/miniconda3/lib/python3.8/site-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.26.5) or chardet (3.0.4) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "

     active environment : conda_env
    active env location : /home/user/miniconda3/envs/conda_env
            shell level : 1
       user config file : /home/user/.condarc
 populated config files : /home/user/.condarc
          conda version : 4.10.3
    conda-build version : 3.21.4
         python version : 3.8.5.final.0
       virtual packages : __cuda=11.2=0
                          __linux=5.4.0=0
                          __glibc=2.27=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /home/user/miniconda3  (writable)
      conda av data dir : /home/user/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/user/miniconda3/pkgs
                          /home/user/.conda/pkgs
       envs directories : /home/user/miniconda3/envs
                          /home/user/.conda/envs
               platform : linux-64
             user-agent : conda/4.10.3 requests/2.24.0 CPython/3.8.5 Linux/5.4.0-64-generic ubuntu/18.04.5 glibc/2.27
                UID:GID : 1572870120:1572870120
             netrc file : None
           offline mode : False
```
</details>

**Comment by ocefpaf** [comment]
`rio-cogeo` pins `rasterio` to an old version that is probably not compatible with newer `fiona` (most probably `fiona`'s deps `proj` and `gdal`). See https://github.com/conda-forge/rio-cogeo-feedstock/blob/master/recipe/meta.yaml#L25

You'll have to use an older `fiona` or wait for a `rio-cogeo` update.

**Comment by ocefpaf** [comment]
Should be fixed in #27.
